### PR TITLE
[R4R] Add BnbDeputyFixedFee param to BEP3 module

### PR DESCRIPTION
**Pull request by denalimarsh**
Resolves https://github.com/Kava-Labs/kava/issues/506

**Problem**: The bep3-deputy specifies an 'other_chain_fixed_fee', a fee paid to the deputy for outgoing swaps (Kava -> BnbChain). If a swap's amount is less than the other_chain_fixed_fee, the bep3-deputy will not relay it. This is usually not a problem as the fixed fee is very, very small (1000bnb, or 10**10-14BNB) but it means micro outgoing swaps would be unfulfillable. An inflexible solution is to set the other_chain_fixed_fee to 0 in the bep3-deputy config, but if the other_chain_fixed_fee is changed then the problem is reintroduced.

**Solution**: This PR adds a BnbDeputyFixedFee parameter that should be 1:1 with the bep3-deputy's config, similar to the BnbDeputyAddress. It was difficult to write specific test cases for the new param - since it's an uint64 we can't pass negative values to it.

**NOTE: ~~base branch is set to dm-fix-rng for ease of review and will be set to master after https://github.com/Kava-Labs/kava/pull/509 is merged.~~

### CODE REVIEW

File: x/bep3/types/params.go [hunk 1]
```diff
@@ -158,6 +169,20 @@ func validateBnbDeputyAddressParam(i interface{}) error {
...
 }
 
+func validateBnbDeputyFixedFeeParam(i interface{}) error {
+	bnbDeputyFixedFee, ok := i.(uint64)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	// Sanity check (impossible for an unsigned integer to be negative)
+	if bnbDeputyFixedFee < 0 {
```
Comment (karzak) [inline x/bep3/types/params.go:179]: nit: this is unnecessary, a better check might be
```suggestion
	if bnbDeputyFixedFee >= uint64(0) - uint64(1) {
```

or just omit it

File: x/bep3/keeper/swap.go [hunk 2]
```diff
@@ -64,11 +64,15 @@ func (k Keeper) CreateAtomicSwap(ctx sdk.Context, randomNumberHash []byte, times
 		}
 		err = k.IncrementIncomingAssetSupply(ctx, amount[0])
 	case types.Outgoing:
+		// Amount in outgoing swaps must be greater than the deputy's fixed fee. Incoming swaps
+		// have already had their fees collected by the deputy during the relay process.
```
Comment (karzak) [inline x/bep3/keeper/swap.go:68]: nit: move comment about Incoming swaps to that case

File: x/bep3/types/params.go (hunk 1 above)
Comment (denalimarsh) [reply x/bep3/types/params.go]: The proposed solution gives error `constant -1 overflows uint64`. Omitted the check entirely as the uint64 datatype already guarantees a non-negative number.

File: x/bep3/keeper/swap.go (hunk 2 above)
Comment (denalimarsh) [reply x/bep3/keeper/swap.go]: updated in latest

**Pull request merged**